docs/quick.md

```diff
@@ -5,11 +5,26 @@
 Before setting up cStor operators make sure your Kubernetes Cluster
 meets the following prerequisites:
 
-1. You will need to have Kubernetes version 1.14 or higher
-2. iSCSI initiator utils installed on all the worker nodes
+1. You will need to have Kubernetes version 1.14 or higher.
+2. iSCSI initiator utils installed on all the worker nodes(If you are using rancher based cluster perform steps mentioned [here](troubleshooting/rancher_prerequisite.md)).
 3. You have access to install RBAC components into kube-system namespace.
 4. You have disks attached to nodes to provision storage.
 
+#### Install iSCSI initiator and load iSCSI modules
+
+| OPERATING SYSTEM | ISCSI PACKAGE         | COMMANDS                                                 |
```

Comment:
Can you shift this table right below step 2 above?

Reply:
Updated